integration-test/src/step_definitions/support/utility/request_builders.js

```diff
@@ -344,6 +344,51 @@ function buildSendRTRequest(gpdSessionBundle, fiscalCode) {
         </Envelope>`;
 }
 
+function buildGetPaymentReq(gpdSessionBundle, fiscalCode) {
+	const brokerCode = gpdSessionBundle.brokerCode;
+	const stationCode = gpdSessionBundle.stationCode;
+	const noticeNumber = `${gpdSessionBundle.debtPosition.iuv1}`;
+	const amount = `${gpdSessionBundle.debtPosition.amount}.00`;
+	return `<soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns:pafn="http://pagopa-api.pagopa.gov.it/pa/paForNode.xsd">
+                        <soapenv:Header />
+                        <soapenv:Body>
+                            <pafn:paGetPaymentReq>
+                                <idPA>${fiscalCode}</idPA>
+                                <idBrokerPA>${brokerCode}</idBrokerPA>
+                                <idStation>${stationCode}</idStation>
+                                <qrCode>
+                                    <fiscalCode>${fiscalCode}</fiscalCode>
+                                    <noticeNumber>3${noticeNumber}</noticeNumber>
+                                </qrCode>
+                                <amount>${amount}</amount>
+                            </pafn:paGetPaymentReq>
+                        </soapenv:Body>
+                    </soapenv:Envelope>`;
+	}
+	
+	function buildGetPaymentV2Req(gpdSessionBundle, fiscalCode) {
+	const brokerCode = gpdSessionBundle.brokerCode;
+	const stationCode = gpdSessionBundle.stationCode;
+	const noticeNumber = `${gpdSessionBundle.debtPosition.iuv1}`;
+	const amount = `${gpdSessionBundle.debtPosition.amount}.00`;
+	return `<soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns:pafn="http://pagopa-api.pagopa.gov.it/pa/paForNode.xsd">
+                        <soapenv:Header />
+                        <soapenv:Body>
+                            <pafn:paGetPaymentReq>
```

Comment:
why paGetPaymentReq and not v2  https://github.com/pagopa/pagopa-api/blob/744eb1a7f33bc59fc0d913477c312fdb3ad5ab6e/wsdl/xsd/paForNode.xsd#L419 ?